src/gadgets/ecc.rs

```diff
@@ -423,6 +423,7 @@ where
     Ok(Self { x, y, is_infinity })
   }
 
+  #[allow(rustdoc::bare_urls)]
```

Comment:
Can't we simply put it in the correct format??

Reply:
Sir, this comment contains a full https link, which does not match the rust requirement. So I add this line.

Now I removed the `#[allow(rustdoc::bare_urls)]` and prefix `https://`, I'm not sure if this is okay.

Reply:
I think the following suggestion should solve it:

Can you try to wrap the link as: `<http://example.org>`?? So use the `<,>`. I think this will remove the lint warning and hence, you can remove the `#allow`

Reply:
Thank you very much.